css-brochure-form/styles.css

```diff
@@ -0,0 +1,75 @@
+* {
+  box-sizing: border-box;
+}
+
+form {
+  margin: auto;
+  max-width: 70%;
+}
+
+.header1 {
+  text-align: center;
+  color: rgb(235 99 92);
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  margin-right: 59px;
+}
+
+.align-center {
+  padding-left: ;
+}
```

Comment:
You should delete this rule, and remove the `align-center` class from any elements that have it, now that it doesn't do anything.